script/tool/lib/src/xctest_command.dart

```diff
@@ -56,9 +59,6 @@ class XCTestCommand extends PackageLoopingCommand {
       'Runs the xctests in the iOS and/or macOS example apps.\n\n'
       'This command requires "flutter" and "xcrun" to be in your path.';
 
-  @override
-  String get failureListHeader => 'The following packages are failing XCTests:';
```

Comment:
Opportunistic cleanup; I left this when doing the initial migration to the new base command to minimize test diff, but there's no reason we need a custom summary header for this command.